src/org/daisy/dotify/formatter/impl/core/TocEntryOnResumedRange.java

```diff
@@ -0,0 +1,28 @@
+package org.daisy.dotify.formatter.impl.core;
+
+/**
+ * Class for the range of toc-entry-on-resumed elements
+ * This class stores a ranges of the form [startRefId,endRefId) or [startRefId,)
+ * 
+ * @author Paul Rambags
+ */
+public class TocEntryOnResumedRange {
+    
+    /* the startRefId refers to the start of the first block in the range, inclusive. May not be null */
+    private final String startRefId;
+    /* the endRefId refers to the start of the last block in the range, exclusive. Can be null */
+    private final String endRefId;
+    
+    public TocEntryOnResumedRange(String startRefId, String endRefId) {
+        this.startRefId = startRefId;
+        this.endRefId = endRefId;
+    }
+    
+    public String getStartRefId() {
+        return startRefId;
+    }
+    
+    public String getEndRefId() {
```

Comment:
An alternative to returning a nullable value here is to return an `Optional<String>`. Makes sense especially if we decide to move this class to the API.

Reply:
Done.